specification/frontdoor/resource-manager/Microsoft.Network/stable/2025-10-01/frontdoor.json

```diff
@@ -0,0 +1,2368 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Front Doors in a Resource Group": {
+            "$ref": "./examples/FrontdoorList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/frontDoors/{frontDoorName}": {
+      "get": {
+        "tags": [
+          "FrontDoors"
+        ],
+        "description": "Gets a Front Door with the specified Front Door name under the specified subscription and resource group.",
+        "operationId": "FrontDoors_Get",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/frontDoorNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FrontDoor"
+            }
+          },
+          "default": {
+            "description": "Front Door error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Front Door": {
+            "$ref": "./examples/FrontdoorGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "FrontDoors"
+        ],
+        "description": "Creates a new Front Door with a Front Door name under the specified subscription and resource group.",
+        "operationId": "FrontDoors_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/frontDoorNameParameter"
+          },
+          {
+            "description": "Front Door properties needed to create a new Front Door.",
+            "in": "body",
+            "name": "frontDoorParameters",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FrontDoor"
+            }
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FrontDoor"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/FrontDoor"
+            }
+          },
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

202 is not recommended. Refer to [https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#creating-replacing-or-updating-resources-asynchronously](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#creating-replacing-or-updating-resources-asynchronously)

Reply:
Note this applies to new RTs only.